core/web/user_controller.go

```diff
@@ -30,10 +30,16 @@ type UpdatePasswordRequest struct {
 	NewPassword string `json:"newPassword"`
 }
 
+var errUnsupportedForAuth = errors.New("action is unsupported with configured authentication provider")
```

Comment:
I was going to suggest wrapping `ErrUnsupported` here, but maybe that doesn't fit for unauthorized? :thinking:

Reply:
In this case I think the standalone makes sense 🙂